### ISSUE TITLE
"Unknown" Exception When Viewing AccessibilityExample RNTester Page

### ISSUE THREAD

**Issue by NickGerleman** [microsoft/react-native-windows#5604]
![image](https://user-images.githubusercontent.com/835219/88711611-47d16600-d0cd-11ea-8b8d-be7341fa2523.png)

Seems potentially new.

**Comment by NickGerleman** [comment]
Hmm, in our 0.63-stable branch this is even worse, just crashing immediately.

**Comment by NickGerleman** [comment]
The crash was outside of web debugging and due to an issue with our impl of accessibilityValue. That was fixed and backported but we still see this issue. Clicking on the LogBox after we see the issue leads to a crash internal to the compositor? Something funky is going on.

**Comment by asklar** [comment]
Filed a new issue for the actual app crash (5700), keeping this bug for the JS crash. That seems to be coming from JSTimers.callTimers ending up timing out but I'm not sure what that means or how to debug it on the JS side

**Comment by acoates-ms** [comment]
This is due to us not having an implementation of the AccessibilityInfo native module.

An error is thrown when calling AccessiblityInfo.fetch.  While that API is deprecated, there is another API isScreenReaderEnabled which it calls, which is not deprecated, and it also throws.

This should only affect people using the AccessiblityInfo APIs.

**Comment by NickGerleman** [comment]
Thanks for looking at that. Feels like this would be annoying for xplat apps, so we should maybe add a stub, but this doesn't feel super serious since it was never implemented before. IIRC AccessibilityInfo JS used to guard against a lack of module, but that seems to be happening less.

**Comment by acoates-ms** [comment]
They are guarding against a lack of the module.  But their guard is to have the promise immediately reject, not to return false.

**Comment by NickGerleman** [comment]
Oh... I think that's what our old stub did as well. Seems like the AccessibilityInfo issue isn't new then?
